radix-engine/src/model/component/executables.rs

```diff
@@ -78,18 +78,43 @@ impl NativeProcedure for ComponentGlobalizeWithOwnerInvocation {
         };
         let owner_badge_bucket: Bucket = api.sys_invoke(mint_invocation)?;
 
-        let mut access_rules = AccessRules::new().default(AccessRule::AllowAll);
-        access_rules.set_access_rule_and_mutability(
+        // TODO: move to sdk
```

Comment:
Need to verify with product :)

Reply:
Not sure if this is a product question. The implementation is problematic as it may conflict with existing access rules that've already included configuration for metadata/royalty etc.

Reply:
Ah I misunderstood the comment then